app/models/dignity_calculator.rb

```diff
@@ -36,6 +36,26 @@ class DignityCalculator
       method: :in_term?,
       score: 2
     },
+    "direct": {
+      method: :direct?,
+      score: 4
+    },
+    "retrograde": {
+      method: :retrograde?,
+      score: -5
+    },
+    "fast": {
+      method: :fast?,
+      score: 2
+    },
+    "slow": {
+      method: :slow?,
+      score: -2
+    },
+    "stationary": {
+      method: :stationary?,
+      score: -1
```

Comment:
It is said stationary planets are more powerful but in terms of pure quantification 0 would be the right score to me. In any case not negative score since by speed they are just standing still (no movement).